.pre-commit-config.yaml

```
@@ -11,7 +11,7 @@ repos:
         exclude: "^(\
             cextern/wcslib/C/flexed/.*|\
             CHANGES.rst|\
-            astropy/extern/jquery/data/js/jquery.dataTables.js|\
+            astropy/extern/jquery/data/js/dataTables.js|\
```

Actual filename in your branch is all lowercase.
```suggestion
            astropy/extern/jquery/data/js/datatables.js|\
```

---

I'm taking care of this in the rebase. It's messy enough to do that. I'm sorry I don't know how to credit you for that in that commit, I hope that's OK.

---

@hamogu you can just add the following at the end of your commit message through `git commit --amend`
```
Co-authored by: P. L. Lim <2090236+pllim@users.noreply.github.com>
```

---

No worries. I don't have to be credited.